### PR TITLE
Implemented tare

### PR DESCRIPTION
**Pull request by Stefan770**

### CODE REVIEW

File: CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs [hunk 1]
```diff
@@ -62,5 +70,21 @@ private void Export_Click(object sented, RoutedEventArgs e)
 
             (DataContext as MeasureViewModel).GenerateReport.Execute();
         }
+
+        private void FillTareCombo()
+        {
+            for (int i = 1; i < 11; i++)
```
Comment (bdelicexecom) [inline CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs]: What number 11 represent? Should it be declared as constant?

File: CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs [hunk 2]
```diff
@@ -62,5 +70,21 @@ private void Export_Click(object sented, RoutedEventArgs e)
...
+            {
+                cmbTare.Items.Add(i);
+            }
+
+            cmbTare.SelectedIndex = 0;
+        }
+
+        private void NumberValidationTextBox(object sender, TextCompositionEventArgs e)
```
Comment (bdelicexecom) [inline CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs]: I think that we already have method for validation text number in Helpers/Validations.

File: CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs [hunk 3]
```diff
@@ -24,7 +26,12 @@ public MeasureView(IDialogService dialogService)
 
         private void OpenDialog(object sender, RoutedEventArgs e)
         {
-            _dialogService.ShowDialog(nameof(AddMeasurementDialog), null, r =>
+            _dialogService.ShowDialog(nameof(AddMeasurementDialog), new DialogParameters
```
Comment (lkovac153) [inline CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs]: This should be done in viewmodel.

File: CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs [hunk 4]
```diff
@@ -78,5 +80,11 @@ private void FillTareCombo()
 
             cmbTare.SelectedIndex = 0;
         }
+
+        private void NumberValidationTextBox(object sender, TextCompositionEventArgs e)
+        {
```
[the remark this answers is not in the document]
Comment (Stefan770) [reply CS.Desktop/Execom.ConnectedScale.Client/Views/MeasureView.xaml.cs]: For textbox masking

**Pull request merged**